test/api-test.js

```diff
@@ -0,0 +1,110 @@
+var expect         = require('chai').expect;
+var path           = require('path');
+var pify           = require('pify');
+var Promise        = require('pinkie');
+var nodeExec       = require('child_process').exec;
+var temp           = require('temp').track();
+var readdirSync    = require('fs').readdirSync;
+var createTestCafe = require('testcafe');
+
+var exec = pify(nodeExec, Promise);
+
+
+var SAUCE_USERNAME   = 'testcafe-saucelabs';
+var SAUCE_ACCESS_KEY = 'e31c3a48-90db-4f03-b2c4-05d40603ee12';
```

Comment:
OMG, why not specified via Travis env vars? Please, revoke key and don't publish it